inframanager/config.yaml

```diff
@@ -57,7 +57,10 @@ InfraManager:
 #        ]
 #        Check https://pkg.go.dev/crypto/tls
 #        for the list of supported ciphersuites.
-  arp-mac: "00:09:00:04:03:14"
+
+# Not required for dpdk
+  arp-mac: "00:0f:00:04:03:14"
+
```

Comment:
Do we need to change the MAC here? Let's keep this to earlier value as there's a higher chance that value will be used.

Reply:
Ok. This value is with the one on the mev.